src/components/FruitForm.jsx

```diff
@@ -0,0 +1,19 @@
+import {FruitCounter} from "./FruitCounter";
+import React from "react";
+
+export function FruitForm() {
+    const [formKey, toggleFormKey] = React.useState(true)
+    return (
+        <form className="fruit-form" key={formKey}>
+            <FruitCounter
+                fruitName="meloenen"/>
+            <FruitCounter
+                fruitName="kiwi's" />
+            <FruitCounter
+                fruitName="bananen"/>
+            <FruitCounter
+                fruitName="appels"/>
+            <button type="button" id="fruit-form-reset"onClick={() => toggleFormKey(formKey => !formKey)}>reset</button>
```

Comment:
Als je de huidige state wil gebruiken, dan kun je beter die gebruiken die meegegeven wordt
```suggestion
            <button type="button" id="fruit-form-reset"onClick={() => toggleFormKey(curr => !curr)}>reset</button>
```
Hierdoor wordt er niet telkens een nieuwe functie aangemaakt in memory, omdat een waarde buiten de functie veranderd wordt